packages/ketcher-react/src/script/render/restruct/ReEnhancedFlag.ts

```diff
@@ -14,34 +14,41 @@
  * limitations under the License.
  ***************************************************************************/
 import ReObject from './ReObject'
-import { Fragment, Box2Abs, Vec2, scale } from 'ketcher-core'
+import { Box2Abs, Vec2, scale, StereoFlag } from 'ketcher-core'
+import ReStruct from './index'
 
 class ReEnhancedFlag extends ReObject {
-  constructor(flag, pos) {
+  private flag: StereoFlag | null
+  private pp: Vec2 | null
+  private path: any
+
+  constructor(flag: StereoFlag | null, pos: Vec2 | null) {
     super('enhancedFlag')
     this.flag = flag
     this.pp = pos
   }
   static isSelectable() {
     return true
   }
-  highlightPath(render) {
+  highlightPath(render: any): any {
```

Comment:
please, use `Render` class for type `render` argument

Reply:
fixed